util/cookieService.js

```diff
@@ -15,15 +15,15 @@ export async function getLoginSession(req) {
   const token = getTokenCookie(req);
 
   if (!token){
-    return;
+    return undefined;
```

Comment:
perhaps we want to return `null` instead of `undefined`? `undefined` tends to refer to something that has not yet been assigned a value. `null` on the other hand is something that definitively has no value. It's not super crucial, both will behave the same honestly.